### ISSUE TITLE
reverse doesn't work anymore

### ISSUE THREAD

**Issue by jordansread** [USGS-R/gsplot#321]
``` r
gs <- gsplot() %>%
   points(1:10, 1:10) %>%
   axis(1, at = seq(0,10,by=0.1),labels=FALSE, tcl=0.15) %>%
   axis(2, reverse=TRUE)
gs
```

![image](https://cloud.githubusercontent.com/assets/2349007/14586929/99d05e92-046d-11e6-8d04-fafc8ea13b5e.png)

no test for this as far as I could tell

**Comment by jordansread** [comment]
Should probably happen within `lim` function, which would use `rev` if the axes is reverse, but this would be way easier and cleaner if `axis` were contained within their appropriate side, re #320

**Comment by lindsayplatt** [comment]
This seems to be fixed in 0.6.1, but still no test
